client/flutter/lib/components/page_button.dart

```diff
@@ -45,7 +45,8 @@ class PageButton extends StatelessWidget {
             children: <Widget>[
               Text(
                 this.title,
-                textScaleFactor: 1.0 + 1.0 * contentScale(context),
+                // textScaleFactorOf is for the device font size
+                textScaleFactor: 1.0 + contentScale(context) * MediaQuery.textScaleFactorOf(context),
```

Comment:
should we consider adding `MediaQuery.textScaleFactorOf(content)` directly into our `contentScale()` method? that way it would apply everywhere we use contentScale, which I think is most of the places we're editing `textScaleFactor`. we'd definitely need to review everything across the app after making that change, but something to consider

Reply:
As long as `contentScale()` is only used for overriding `textScaleFactor`,

Reply:
never overriding `textScaleFactor` is the way to go in my opinion.

Reply:
Yah, we should not be setting `testScaleFactor` for this as that defeats the system adaptive text size.  We should have used a `fontSize` instead.  *However* these thing are really more like titles and and I don't know that it is expected that UI elements change text size with the adaptive text anyway.  Allowing these to change size is going to complicate @clementmouchet 's work on getting these button/section titles to fit in all languages.  

We should get some input on whether these particular UI elements would be expected to change size with the adaptive text or whether only "content" text should adjust size automatically.  If they need to be adaptive the simplest solution is to remove the `textScaleFactor` and just use a font that matches the current size (the scaling factor can be kept, just used on the font size instead).  

Longer term I think we should switch to the `auto_text_size` package and (per discussion with @clementmouchet ) perhaps include some per-language hints about size, weight, and max lines within the ARB files in the form of annotations.

Reply:
I suggest the latter. I think you will be OK for the six locales the app supports. Russian may wrap in a few places but that is OK as long as its not in a logo.

Reply:
Ok, after discussing with @dleuck it sounds like we *should* allow these button/section titles to update with the adaptive text.  So I think the solution for short term is to switch from the `textScaleFactor` to `fontSize` just trying to match the existing (and using our `contentScale()` on the `fontSize` just as we did on the `textScaleFactor`.

Reply:
Can I create a new pr that uses `auto_text_size`? @patniemeyer

Reply:
@patniemeyer FYI: https://github.com/WorldHealthOrganization/app/issues/884#issuecomment-609462881